.taskcluster.yml

```diff
@@ -35,20 +35,20 @@ tasks:
         taskId: ${taskGroupId}
         taskGroupId: ${taskGroupId}
         schedulerId: taskcluster-github
-        provisionerId: aws-provisioner-v1
-        workerType: github-worker
+        provisionerId: proj-git-cinnabar
+        workerType: ci
         retries: 5
         scopes:
           $flatten:
             - assume:repo:github.com/${e.base_login}/${e.base_repo_name}:decision-task
```

Comment:
Is that still valid?

Reply:
It is -- it's added in https://github.com/mozilla/community-tc-config/pull/37